### DISCUSSION TITLE
How do I update the forge modloader when using the old manual Curseforge server type?

### DISCUSSION THREAD

**Discussion by hamcoremusic** [itzg/docker-minecraft-server#2890] [Q&A] [answered]
So after trying to get BMC4 to work through `AUTO_CURSEFORGE` and `MODRINTH` I decided to revert to the manual Curseforge option and it worked way better. The server pack just has a lot tied to it that the auto downloaders didn't work with.

Now v28 of BMC4 came out and it requires a bump in the Forge mod loader version. How would I go about this? Would I just change the pack name to the new version and let it run it's course?

My only issue with making the container unpack the server is it might overwrite my `variables.txt` and server properties. Can I safely remove those out of the pack and it still remains functional?

**Comment by hamcoremusic** [comment]
Or can I just change the Forge version to the one I need in `variables.txt`?

**Reply by itzg** [reply]
The **old** approach is deprecated for several reasons but mainly because every server modpack is a little different. You'll need to investigate that variables file and the modpack supplied script to see if that solves your scenario.

**Reply by hamcoremusic** [reply, marked as answer]
Looks like I just has to make sure I brought over the `start.sh` script and copy my custom flags to the `variables.txt` file. It reads the version it needs from the variable file so it auto-installed using their script
